### ISSUE TITLE
Problem with displaying date of month in the system tray

### ISSUE THREAD

**Issue by gitthepie** [mikeberger/borg_calendar#180]
After a recent upgrade from Linux Mint 20 to 21, I noticed that system tray functionality isn't working correctly. I was on an older 1.10 version and upgraded to 1.10.03, but the problem persists. The system tray icon used to show the date of the month, but now shows just a black box. The date does show, temporarily, in a floating box at the top left of the screen. The actual system tray icon is just a black box.

What might be causing this problem? I've attached 3 pictures.

This one shows shows the floating date icon. The red part is my wallpaper:
![Top left of the screen](https://user-images.githubusercontent.com/57020624/208325728-0e2d682f-e6cf-4b86-bfae-591ac2139a81.jpg)

This one shows what the system tray icon looks like. The black square is what used to be the Borg date or icon:
![Bottom right](https://user-images.githubusercontent.com/57020624/208325743-506247ac-0c65-4f73-b95b-aec611f20b93.jpg)

This one shows what the floating date icon looks like when I turn off the "Show Date in System Tray" option to try to get around the problem:
![Borg icon](https://user-images.githubusercontent.com/57020624/208330881-f01f7530-c1c4-4e2e-88f8-8dcd1681ea89.jpg)

**Comment by gitthepie** [comment]
Thanks for the quick response Mike! It's not a big deal; I just wanted to know the cause. I thought at first it was because of the way I customized and locked my taskbar, but I guess this one is out of my control. The calendar is too important for me to pause usage just because there's a minor bug. I'm happy to wait this one out. In fact, the reason the taskbar is gray rather than a pure black is because I'd already accepted that the icon would be blank until a fix is released. I changed my theme to better see the blank icon!
